config/defaults.json

```diff
@@ -0,0 +1,135 @@
+{
+  "cli_wallet_connection": "",
+  "db": {
```

Comment:
(And even if we don't, we should use prepared statements with placeholders instead of string interpolation/concatenation.)